Makefile

```diff
@@ -1,10 +1,9 @@
 
-
 build:
-	docker build -t firefox .
+	docker build --build-arg FIREFOX_VERSION=$(shell cat VERSION) -t firefox:$(shell cat VERSION) .
```

Comment:
I don't see why I shouldn't work on docker.io. The only problem is that you need the correct namespacing for the docker repository. In your case it would be `splattael/firefox` and everything will work. As far as I know when you use `Public Automated Builds` at docker.io it will checkout the repo and use `.dockerignore` and `Dockerfile` to build your Image. The Makefile has nothing to do with it.